From: =?utf-8?q?IOhannes_m_zm=C3=B6lnig?= <umlaeute@debian.org>
Date: Tue, 29 Jan 2019 11:31:36 +0100
Subject: search /usr/lib/pd/extra/ for externals

since we install into /usr/lib/puredata/extra, the ordinary install path for
externals (/usr/lib/pd/extra) won't get searched automatically; so we need to
add it manually
---
 src/s_path.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/s_path.c b/src/s_path.c
index 0066d6f..ea4da62 100644
--- a/src/s_path.c
+++ b/src/s_path.c
@@ -264,6 +264,10 @@ void sys_setextrapath(const char *p)
 #endif
     /* add built-in "extra" path last so its checked last */
     STUFF->st_staticpath = namelist_append(STUFF->st_staticpath, p, 0);
+    /* DEBIAN: add common path for all pd-flavours very last, so puredata
+     * specific externals get higher priority */
+    STUFF->st_staticpath = namelist_append(STUFF->st_staticpath, "/usr/lib/pd/extra", 0);
+
 }
 
     /* try to open a file in the directory "dir", named "name""ext",
